modin/engines/base/frame/data.py

```diff
@@ -204,6 +211,21 @@ def _set_columns(self, new_columns):
                 self._dtypes.index = new_columns
         self._apply_index_objs(axis=1)
 
+    def _set_axis(self, axis, new_axis):
```

Comment:
I prefer not to add this, you can use `self.axes[axis] = new_axis`

Reply:
`self.axes` will return us `[self.index, self.columns]`, roughly say just an array of values, so assignment like `self.axes[axis] = new_axis` will not do any effect, look at that reproduce:
```python
import pandas

df = pandas.DataFrame([1,2,3])
print(df.axes[0]) # RangeIndex(start=0, stop=3, step=1)
df.axes[0] = pandas.Index(['a', 'b', 'c'])
print(df.axes[0]) # RangeIndex(start=0, stop=3, step=1)
```

Reply:
Not enough coffee yet 😄